junit-vintage-engine/src/main/java/org/junit/vintage/engine/discovery/IsPotentialJUnit4TestClass.java

```diff
@@ -10,16 +10,20 @@
 
 package org.junit.vintage.engine.discovery;
 
+import static org.apiguardian.api.API.Status.INTERNAL;
 import static org.junit.platform.commons.util.ReflectionUtils.isAbstract;
 import static org.junit.platform.commons.util.ReflectionUtils.isInnerClass;
 import static org.junit.platform.commons.util.ReflectionUtils.isPublic;
 
 import java.util.function.Predicate;
 
+import org.apiguardian.api.API;
+
 /**
  * @since 4.12
  */
-class IsPotentialJUnit4TestClass implements Predicate<Class<?>> {
+@API(status = INTERNAL, since = "5.8", consumers = "org.junit.vintage.**")
```

Comment:
Did we decide on Ant-style pattern matching for wildcards in the `consumers` attribute?

I honestly don't recall.

Reply:
I don't think we made a decision but I figured this would at least be unambiguous.

Reply:
Fair enough. We can certainly leave it like for now and change it later if necessary based on tooling support or clearer specifications in API Guardian.